app/models.py

```diff
@@ -0,0 +1,30 @@
+from marshmallow import Schema, fields
+import random
+
+class User():
+
+    def __init__(self, username, department, email, password):
+        self.user_id = random.randint(1,100)
```

Comment:
Are you certain that by the time you have 100+ users there won't be users with the same user id??

Reply:
I'll change that, it was only for demonstration purposes